sobject.go

```diff
@@ -104,7 +105,10 @@ func (obj *SObject) Create() *SObject {
 	}
 
 	url := obj.client().makeURL("sobjects/" + obj.Type() + "/")
+	fmt.Println("URL: " + url)
```

Comment:
I understand this fmt.Println() is more for debugging purpose, right? In that case, would it be better to use log.Println() instead? I know the project doesn't have a proper logging mechanism anyway, but fmt.Println() may be too harsh since it goes to stdout directly.

Reply:
Good catch, Wenchao!  I actually didn't intend to leave those in so I've removed them altogether. Thanks.

Reply:
Thanks, Xavier!